integration-tests/lints/known_attributes.mj

```diff
@@ -0,0 +1,15 @@
+//?!noinline
```

Comment:
I really don't understand why we have to copy the cryptic syntax of rust. `globally disable/enable LINTNAME` instead of `?!` would be preferable. Same for non-global attributation